tools/ido_recomp/README.md

```diff
@@ -0,0 +1,15 @@
+# Static recomp of IRIX programs
```

Comment:
If we are going to have ido_recomp in the repo, shouldn't it be a better idea to add it as a subrepo?

Reply:
Its probably better practice to do a subrepo but recomp hasn't changed in a really long time so I think its OK for now.